src/rust/engine/workunit_store/src/lib.rs

```diff
@@ -425,23 +426,24 @@ impl HeavyHittersData {
     }
   }
 
-  fn heavy_hitters(&self, k: usize) -> HashMap<SpanId, (String, Option<Duration>)> {
+  fn heavy_hitters(&self, k: usize) -> HashMap<SpanId, (String, SystemTime)> {
```

Comment:
@stuhood Do you think this change makes sense in isolation? And even if `prodash` is a ways off? I think it's nicer to have this return the start time, as callers an calculate duration if they want (but can't really get start time from duration)

Reply:
Yea, this could be landed independently.

Reply:
I'll get to work. I'll be excited to make this PR smaller :)

Reply:
https://github.com/pantsbuild/pants/pull/14240